test/compress/object.js

```diff
@@ -22,7 +46,31 @@ getter_setter: {
         var get = "bar";
         var a = {
             get,
-            set: "foo"
+            set: "foo",
+            get bar() {
+                return this.get;
+            },
+            get 5() {
+                return "five";
+            },
+            get 0x55() {
+                return "five five";
+            },
+            get "five"() {
```

Comment:
That could be optimized indeed.

Also note that this code may not work in chrome and that there aren't test262 tests for this yet, although it is in progress (https://github.com/tc39/test262/issues/703 and https://github.com/tc39/test262/pull/706).

But this shouldn't stop us implementing this, as this code works in firefox and is in line with specs.